lib/widget/webview/webview.dart

```diff
@@ -98,3 +101,17 @@ List<Map<String, dynamic>> getListOfMap(list) {
   }
   return [];
 }
+
+Map<String, String> parseYamlMap(value) {
+  Map<String, String> result = {};
+  if (value is YamlMap) {
+    YamlMap yamlMap = value;
+    for (var i in yamlMap.entries) {
```

Comment:
just a small note - can you please use a descriptive variable name instead of i. I see this in your other PRs as well. Just call it entry in this case. i is the convention for the counter when you are iterating with an index.